app/models/financialDetails/YourSelfAssessmentChargesViewModel.scala

```diff
@@ -23,6 +23,7 @@ import services.DateServiceInterface
 import java.time.LocalDate
 
 case class YourSelfAssessmentChargesViewModel(hasOverdueOrAccruingInterestCharges: Boolean,
+                                              hasChargesDueWithin30Days: Boolean,
```

Comment:
this can be resolved by what we have in  `whatYouOweChargesList: WhatYouOweChargesList ` ?